android/java/org/chromium/base/Callbacks.java

```diff
@@ -0,0 +1,248 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Comment:
are we going to fix our code in a follow-up?

Reply:
yes, this would be advisable, created a follow up issue https://github.com/brave/brave-browser/issues/35872

Reply:
cc: @samartnik